### ISSUE TITLE
Save cover in data

### ISSUE THREAD

**Issue by Frulko** [ktruckenmiller/node-shairport-metaparser#1]
Hi, 
first thanks for your work :) It's very helpful for my project. 
I just want to know if you have any idea about how can we save the image cover sent in metadata? 

thanks.

**Comment by ktruckenmiller** [comment]
In my script I'm querying iTunes for that.

**Comment by Frulko** [comment]
Yes i saw that ^^ is it hard to save this image ? that's why you use iTunes ? 
Maybe an encrypted image data.

**Comment by ktruckenmiller** [comment]
Yes it's very inconsistent and it is based on the user sending it.  This makes it a bit more robust. You could probably save the file using the fs library or something else that downloads images from urls

**Comment by Frulko** [comment]
Ok thanks i'll use iTunes. :) But just for the challenge, I tried to save the raw data in PICT code as an image (just write data and add .jpg extension) but it seems not working.. have you did any research about how the data was formed ?

**Comment by ktruckenmiller** [comment]
I believe shairport sync has a setting where you can store the image.  It comes in late, and you have to check the directory periodically to see if it exists. I'm not super sure because it was a long time ago - but you should check that out and then modify the node script accordingly
